src/screens/ConnectSignUpScreen.js

```diff
@@ -120,6 +122,23 @@ export default class ConnectSignUp extends React.Component {
     }
   }
 
+  getFormOptions() {
```

Comment:
I think this was already in your other PR and I think its right in the other (now merged) PR and not right here. Could you possibly rebase with master?